packages/editor-ui/src/components/mixins/pushConnection.ts

```diff
@@ -173,12 +173,12 @@ export const pushConnection = mixins(
 					return false;
 				}
 
-				if (['nodeExecuteAfter', 'nodeExecuteBefore'].includes(receivedData.type)) {
+				if (receivedData.type === 'nodeExecuteAfter' || receivedData.type === 'nodeExecuteBefore') {
```

Comment:
preferring the previous syntax tbh

```suggestion
				if (['nodeExecuteAfter', 'nodeExecuteBefore'].includes(receivedData.type)) {
```

Reply:
I agree with the comment, but discriminated unions require the syntax `var === 'str'` in order to work - I only changed the `includes` syntax because TS was not narrowing down the type with `includes`.